### ISSUE TITLE
Infinite recursive function

### ISSUE THREAD

**Issue by roigcarlo** [KratosMultiphysics/Kratos#970]
Load and save functions in `multipoint_constraint_data.hpp" are calling themselves making it a infinite recursion loop. This will crash.

```C++
virtual void save(Serializer &rSerializer) const
{
    KRATOS_SERIALIZE_SAVE_BASE_CLASS(rSerializer, MpcData);
}

virtual void load(Serializer &rSerializer)
{
    KRATOS_SERIALIZE_LOAD_BASE_CLASS(rSerializer, MpcData);
}
```

**Comment by loumalouomega** [comment]
In fact, in my experience, the MPC crashes randomly in the tests

**Comment by roigcarlo** [comment]
Well I haven't test it to be honest, I am just reporting win errors (with a little bit more info). Anyway issue was not for you, I misplaced it sorry

**Comment by pooyan-dadvand** [comment]
I think I have report this in my code review....

**Comment by adityaghantasala** [comment]
Sorry for bad code. It probably is a leftover from copy-paste operation.

@loumalouomega  this random crashing is fixed in #972 

@roigcarlo @pooyan-dadvand  the serializer of Kratos does not have a method for std::map / std::unordered_map . Untill such a utility is available (boost serializer is an option here) I am commenting out the save and load functions of this class.

**Comment by adityaghantasala** [comment]
On the second thought I implemented a custom save and load functions for the MpcData class. I am opening a pull request for review.

**Comment by roigcarlo** [comment]
Closed via #977
